### ISSUE TITLE
Alguém vai de Lit?

### ISSUE THREAD

**Issue by LiberQuack** [braziljs/forum#17]
Galera, alguém está se aventurando com [LitElement](https://github.com/Polymer/lit-element) e/ou outra biblioteca de WebComponents?

Achei o LitElement parecido com o react, porém por não utilizar do famoso **VDOM** me chama mais atenção... o [lit-html](https://github.com/Polymer/lit-html) que é responsável pela renderização possui apenas **~2kb**.

**Comment by renanbastos93** [comment]
Opa, não conhecia está biblioteca, achei interessante pelo que você salientou. Irei dar uma olhada!

**Comment by felipenmoura** [comment]
Sim, Lit é bem legal mesmo. Super simples, bem leve intuitivo :)
